test/cli.test.js

```diff
@@ -88,7 +88,7 @@ describe('cli', () => {
     });
 
     it('should not show related commands on commands that have none', () => {
-      cli(['login', '--help']).then(output => {
+      cli(['swagger', '--help']).then(output => {
```

Comment:
Is there a reason for this change?

Reply:
`login` now has related commands, where as `swagger` has none.

Reply:
Makes sense.